<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Estacionamiento</class>
 <widget class="QMainWindow" name="Estacionamiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estacionamiento</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Introduzca su número de cliente para acceder</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNoCliente">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="accederEst">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAcceder">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Boton coulto </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="txtAdmin">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPswrd">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No. Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Contraseña</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIngresar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingresar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
